### DISCUSSION TITLE
opening driver

### DISCUSSION THREAD

**Discussion by adrianc420** [OpenTabletDriver/OpenTabletDriver#1110] [Q&A] [answered]
hello, 

i have a slight problem. well not really but whenever i open the driver from anywhere but the original file location folder. it will just 
![opentabletdriver](https://user-images.githubusercontent.com/83922380/118995525-1f2d9c00-b97f-11eb-8af7-2d520f8ded5d.png) say that.

**Comment by X9VoiD** [comment, marked as answer]
The daemon and the ux must be on the same folder. If you don't want that then you should manually launch daemon.

**Reply by adrianc420** [reply]
so if i wanted it to startup at boot i should make deamon startup instead of the ux

**Reply by X9VoiD** [reply]
If you want it to start at boot then follow this:
![image](https://user-images.githubusercontent.com/10338031/119157725-0eb51880-ba88-11eb-893e-f1e224bbcdc1.png)
